warehouse/warehouse.go

```diff
@@ -139,6 +139,8 @@ type ErrorResponseT struct {
 func Init4() {
 	loadConfig()
 	pkgLogger = logger.NewLogger().Child("warehouse")
+
+	setMaxParallelLoads()
```

Comment:
Why moving it to the end of the inits, why not at the start of Init1. What if some of the inits require these configs.